internal/security/fileprovider/defaults.go

```diff
@@ -0,0 +1,44 @@
+//
+// Copyright (c) 2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License. You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License
+// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+// or implied. See the License for the specific language governing permissions and limitations under
+// the License.
+//
+// SPDX-License-Identifier: Apache-2.0'
+//
+
+package fileprovider
+
+func makeDefaultTokenPolicy(serviceName string) map[string]interface{} {
+	protectedPath := "secret/edgex/" + serviceName + "/*"
```

Comment:
Should we validate the input `serviceName` is something non-empty?

Reply:
The ultimate input is coming from token-config.json where the service name is a key in a key-value map.  It is invalid JSON for this to be empty.  Making the assumption that this is a non-malicous path.

Reply:
ok, sounds good to me then